### PR TITLE
EVG-16058 Display tickets filed with the created_tickets URL in the build baron

### PR DESCRIPTION
**Pull request by malikchaya2**
[EVG-16058](https://jira.mongodb.org/browse/EVG-16058)

### Description 
Display tickets that were filed using the custom ticket api endpoint. 
image

### Testing 
Tested locally

### CODE REVIEW

File: public/static/plugins/buildbaron/js/task_build_baron.js [hunk 1]
```diff
@@ -66,6 +66,14 @@ mciModule.controller('TaskBuildBaronCtrl', function($scope, $http, $window, $int
       });
   }
 
+  $scope.getCustomCreatedTickets = function() {
+    $http.get('/plugin/buildbaron/custom_created_tickets/' + $scope.taskId).then(
+      function(resp) {
+        $scope.custom_created_tickets = resp.data;
+        console.log("Custom tickets: " + JSON.stringify(resp.data));
```
Comment (ablack12) [inline public/static/plugins/buildbaron/js/task_build_baron.js:73]: Leftover debugging?

File: service/ui_plugin_build_baron.go [hunk 2]
```diff
@@ -109,6 +110,23 @@ func (uis *UIServer) bbGetCreatedTickets(w http.ResponseWriter, r *http.Request)
 	gimlet.WriteJSON(w, results)
 }
 
+func (uis *UIServer) bbGetCustomCreatedTickets(w http.ResponseWriter, r *http.Request) {
+	taskId := gimlet.GetVars(r)["task_id"]
+	var results []annotations.IssueLink
+
+	annotations, err := annotations.FindByTaskId(taskId)
```
Comment (ablack12) [inline service/ui_plugin_build_baron.go:117]: Do we not want to filter this by task execution?

File: public/static/plugins/buildbaron/js/task_build_baron.js (hunk 1 above)
Comment (malikchaya2) [reply public/static/plugins/buildbaron/js/task_build_baron.js:73]: yes, thank you!

File: service/ui_plugin_build_baron.go (hunk 2 above)
Comment (malikchaya2) [reply service/ui_plugin_build_baron.go:117]: I didn't because build baron created tickets don't.

Comment (ablack12) [reply service/ui_plugin_build_baron.go:117]: I'm still a bit suspicious but I suppose we can see if it becomes a problem; I'm not sure why we would or wouldn't use execution really.

**Review by ablack12 — APPROVED** [review]

**Comment by malikchaya2** [comment]
evergreen retry

**Pull request merged**